package.json

```diff
@@ -330,7 +330,7 @@
     "stylelint-config-standard": "^26.0.0",
     "stylelint-config-standard-scss": "^5.0.0",
     "swagger-cli": "^4.0.4",
-    "webpack": "^5.73.0",
```

Comment:
maybe a question for the team, but can we remove `webpack` now that we're using `esbuild` or are we still using it for something?

Reply:
@codyseibert Any reason you can think of that we shouldn't remove `webpack`?

Reply:
Looks like we still use for the API